### PR TITLE
docs: Update the screenshots in the Dashboard and Administration Sections

### PR DESCRIPTION
**Pull request by jfermi**
This PR...

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/testkube-pro/articles/environment-management.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 # Environment Management
 
-Environment in Testkube is related to the Testkube agent, which is responsible for sending
-test insights to Testkube Pro and for managing your Kubernetes related cluster resources.
+A Testkube environment is managed by a Testkube Agent running in your Kubernetes Cluster - read more [here](../articles/architecture.md).
```
Comment (olensmar) [inline docs/testkube-pro/articles/environment-management.md:3]: please put this line in a note and add the line "An environment is a grouping of users, workflows, results, triggers, and webhooks." before it

Comment (jfermi) [reply docs/testkube-pro/articles/environment-management.md]: Updated.

File: docs/testkube-pro/articles/environment-management.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
 # Environment Management
 
-Environment in Testkube is related to the Testkube agent, which is responsible for sending
-test insights to Testkube Pro and for managing your Kubernetes related cluster resources.
+:::note
+An environment is a grouping of users, workflows, results, triggers, and webhooks. A Testkube environment is managed by a Testkube Agent running in your Kubernetes Cluster - read more [here](../articles/architecture.md).
```
Comment (olensmar) [inline docs/testkube-pro/articles/environment-management.md:4]: almost :D - the "An environment is a grouping of users, workflows, results, triggers, and webhooks." statement should be before/outside the note

Comment (jfermi) [reply docs/testkube-pro/articles/environment-management.md]: Sorry - done.

**Review by olensmar — APPROVED** [review]
the dashboard does so much more than this - but let's merge this and plan the next iteration of the dashboard docs..

**Pull request merged**